amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -130,28 +147,34 @@ enum ColonyType {
   METACOLONY
 }
 
+enum SortingMethod {
+  BY_HIGHEST_REP
+  BY_LOWEST_REP
```

Comment:
I know that this comes from the dapp but now it seems that we have the camel case for enums so I would update

Reply:
We're using CamelCase for names but the members still seem to be all caps - have a look at `TokenType`

Reply:
Well, then we are not consistent with the FE, because I checked there and it's camel case. I don't really mind either way but I think it would be great to have consistency across everything.